\documentclass{article}


\usepackage{arxiv}
\usepackage{authblk}
\usepackage{cite}
\usepackage[utf8]{inputenc} % allow utf-8 input
\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
\usepackage{hyperref}       % hyperlinks
\usepackage{url}            % simple URL typesetting
%\usepackage[hyphens]{url}
\PassOptionsToPackage{hyphens}{url}\usepackage{hyperref}
\usepackage{booktabs}       % professional-quality tables
\usepackage{amsfonts}       % blackboard math symbols
\usepackage{nicefrac}       % compact symbols for 1/2, etc.
\usepackage{microtype}      % microtypography
\usepackage{lipsum}
\usepackage{subfig}
\usepackage{graphicx}
\graphicspath{ {./images/} }


\title{Contributions of the Petabyte Scale Sequence Search Codeathon toward efforts to scale sequence-based searches on the SRA}


\author[1]{A. Author}
\author[2]{B. Author}
\author[3]{C. Author}
%\author[2]{Corresponding Author\thanks{email@2nduniversity.com}}
\affil[1,2]{Author Affiliation 1}
\affil[3]{Author Affiliation 2}
{
    \makeatletter
    \renewcommand\AB@affilsepx{: \protect\Affilfont}
    \makeatother

    \affil[ ]{Email ids}

    \makeatletter
    \renewcommand\AB@affilsepx{, \protect\Affilfont}
    \makeatother

    \affil[1]{aa@x.gov}
    \affil[2]{ba@y.gov}
    \affil[1,3]{ca@z.gov}
}

\begin{document}
\maketitle
\begin{abstract}

The volume of biological data being generated by the scientific community is growing exponentially, reflecting technological advances and research activities. This increase in available data has great promise for pushing scientific discovery but also introduces new challenges that scientific communities need to address. The National Institutes of Health’s (NIH) Sequence Read Archive (SRA), which is maintained by the National Library of Medicine’s National Center for Biotechnology Information (NCBI), is a rapidly growing public database that researchers use to improve scientific discovery across all domains of life. As part of the Science and Technology Research Infrastructure for Discovery, Experimentation, and Sustainability (STRIDES) Initiative, over 40 petabytes of “next generation” (raw and SRA-formatted) sequencing data is accessible to anybody via two cloud service providers.

To help address the challenges of conducting large-scale analysis of omic data in the SRA and similar databases, the Department of Energy (DOE) Office of Biological and Environmental Research (BER), the NIH Office of Data Science Strategy (ODSS), and NCBI, held a virtual codeathon 'Petabyte Scale Sequence Search: Metagenomics Benchmarking Codeathon' on September 27 - Oct 1 2021, to address emerging Solutions in Petabyte Scale Sequence Search. The codeathon  attracted experts from national laboratories including the Los Alamos National laboratory (LANL), research institutions including the Joint Genome Institute (JGI) and students from universities across the world to develop benchmarking approaches to address challenges in conducting large-scale analyses of metagenomic data.

\end{abstract}


% keywords can be removed
%\keywords{First keyword \and Second keyword \and More}


\input{introduction}
\input{sequence_search}
\input{codeathon}

%\clearpage

\section*{Acknowledgments}
This work was supported in part by the National Center for Biotechnology Information of the National Library of Medicine (NLM), National Institutes of Health.



%\bibliographystyle{unsrt}  
%\bibliography{references}  %%% Remove comment to use the external .bib file (using bibtex).
%%% and comment out the ``thebibliography'' section.


%%% Comment out this section when you \bibliography{references} is enabled.


\bibliography{references}

%This defines the bibliographies style. Search online for a list of available styles.
\bibliographystyle{abbrv}


\end{document}
